lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -829,6 +831,7 @@ public function where($predicates)
     public function andWhere($where)
     {
         $args  = func_get_args();
+        $args  = self::filterEmptyArgs($args);
```

Comment:
As far as I understand, we need to cover both branches with the `with()` and `::and()` because the latter used to be implemented by `->addMultiple()` → `->add()` as well. Would it be possible to cover it with a test as well?

Once implemented, please replicate the test for the rest of the changed methods.

Reply:
Tests added!

Reply:
Are you sure the four added tests cover the four updated methods with two execution branches in each?

Reply:
There are 8 tests added!

Reply:
I'm sorry. I think I saw what I expected to see.